packages/program-boilerplate/src/utils.ts

```diff
@@ -133,3 +136,56 @@ export function numToEquality(num: number): string {
       return 'eq';
   }
 }
+
+export function getTriggerSchema(body: ProgramTriggerBody): object[] {
```

Comment:
I'm not really sure what the purpose of this function is

Reply:
I have added a comment before it to serve as an explanation